packages/connectivity_plus/connectivity_plus/ios/Classes/PathMonitorConnectivityProvider.swift

```diff
@@ -19,6 +19,11 @@ public class PathMonitorConnectivityProvider: NSObject, ConnectivityProvider {
         // .wiredEthernet is available in simulator
         // but for consistency it is probably correct to report .wifi
         return .wifi
+      } else if path.usesInterfaceType(.other) {
+        // path.status == .satisfied means that the network is available
+        // and .other is available in simulator
+        // for consistency, it would probably be correct to report .vpn
+        return .vpn
```

Comment:
I am not convinced about this. Of course, "none" is not correct either, but as far as I understand we don't have the certainty that this is a VPN.

Maybe adding another type `other` would be better?

Reply:
I wouldn't want to break the common interface with the android version. What's here, what's in android, vpn doesn't guarantee network connectivity.

It says "other" in the documentation:`The network interface type used for communication over virtual networks or networks of unknown types.`

Anyway, I agree with you that "other" is not "vpn", waiting for a final decision from you.

Reply:
> The network interface type used for communication over virtual networks or networks of unknown types.

Thanks for pointing this out! VPNs do indeed appear as "other", so it kind of makes sense as well.

Any thoughts @nohli @vbuberen ?

Reply:
> Anyway, I agree with you that "other" is not "vpn", waiting for a final decision from you.

@miquelbeltran I don't get the last sentence. I would be fine with changing line 26 to `other`.
If it should be added *additionally* to `vpn`, which argument would be passed to `usesInterfaceType()` for `other`?

Reply:
@nohli @miquelbeltran should I change `vpn` to `other` for iOS/MacOS?

Reply:
No need to tag people all the time. Those, who are in a thread will get an email anyway.

Yes, agree with suggestion from Miguel to have `other` returned for such cases.

Reply:
Okay, I changed it to `other`.